html-templates/register/register.tpl

```diff
@@ -73,6 +73,13 @@
             </div>
             <div class="control-group">
                 <label class="control-label"></label>
+                 <div class="controls">
+                  {ReCaptcha::widget()}
+                  </div>
+                  </div>
```

Comment:
@RowdyChildren indentation makes it unclear if the opening/closing divs match up